fastlane/Fastfile

```diff
@@ -86,21 +86,34 @@ end
 # `readonly`:
 #    if true  - only fetch existing certificates and profiles, don't upgrade from AppStoreConnect
 #    if false - read list of devices from AppStoreConnect, and upgrade the provisioning profiles from it
-def build_ios_adhoc(readonly)
+def build_ios_adhoc(readonly: false, pr_build: false)
+  # PR builds should appear as a separate App on iOS
+  scheme = pr_build ? 'StatusImPR' : 'StatusIm'
+  app_id = pr_build ? 'im.status.ethereum.pr' : 'im.status.ethereum'
```

Comment:
Can we maybe add a comment also explaining the `pr_build` argument the same way `readonly` is?